.github/workflows/scripts/trigger_tests_on_label.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import os
+from urllib.parse import quote
+from urllib.request import Request, urlopen
+
+LABEL = "Trigger: tests"
```

Comment:
I would maybe make this more explicit that it's sensitive tests requiring secrets

Reply:
Changed to `"Trigger: tests using secrets"`.